com.musala.simple.students.spring/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+server.port: 1234
```

Comment:
Considering the hard-coded localhost address in the controller - if the suggested solutions don't work, you can take the address from this configuration as a fallback mechanism.

Reply:
Ok, I did that :)